docs/source/WritingTests.rst

```diff
@@ -1034,6 +1034,48 @@ Notice the ``test3`` was not skipped because the provided condition was
 not ``False``.
 
 
+Cancelling Tests
+================
+
+The only supported way to cancel a test and not negatively impact the
+job exit status (unlike using `self.fail` or `self.error`) is by using
+the `self.cancel()` method. The `self.cancel()` can be called only
+from your test methods. Example::
+
+    #!/usr/bin/env python
+
+    from avocado import Test
+    from avocado import main
+
+
+    class CancelTest(Test):
+
+        """
+        Example test that cancels the current test from inside the test.
+        """
+
+        def test(self):
+            self.cancel("This should end with CANCEL.")
```

Comment:
This is a very straightforward example, but maybe a more real world example would be even more didactic.  For example, the original use case was checking if something provided (configured) by params was available, and if not, the test would not go further (would be canceled).